packages/terra-worklist-data-grid/src/proptypes/rowShape.js

```diff
@@ -0,0 +1,15 @@
+import PropTypes from 'prop-types';
+import cellShape from './cellShape';
+
+const rowShape = PropTypes.shape({
+  /**
+   * String that specifies the row height. Any valid CSS height value is accepted.
+   */
+  height: PropTypes.string,
```

Comment:
The row height should be part of the main worklist.  It will not vary per row.  It would not be part of the row shape.

Reply:
@chrismichalewicz  & @eawww, could you clarify if all rows of the grid will have same height or can row height vary? We assumed that they could vary based on what we saw in Clinical data grid. We can refactor if this assumption is incorrect.